docs/helm/kind-deployment.md

```diff
@@ -56,17 +56,26 @@ Wait for the Kind cluster to be created. This may take a few minutes.
 
 ## Step 3: Install patched ingress-nginx
 
-Follow the instructions for
-[deploying nginx ingress on KinD](https://kind.sigs.k8s.io/docs/user/ingress/#ingress-nginx).
+Install the ingress-nginx controller namespace:
+
+```bash
+helm upgrade --install ingress-nginx ingress-nginx \
+--repo https://kubernetes.github.io/ingress-nginx \
+--namespace ingress-nginx --create-namespace
+```
```

Comment:
see https://github.com/kubernetes-sigs/kind/issues/1693#issuecomment-1768116494

I don't expect to be working without any change, otherwise there was no need to have a "kind" flavor of ingress-nginx